src/core/deployer.sol

```diff
@@ -70,11 +71,11 @@ contract ShelfFab {
     }
 }
 
-contract DeskFab {
-    // note: this is the mock Desk, which will interface with the lender/tranche side of Tinlake, and does not require auth for now.
-    function newDesk(address shelf_, address token_) public returns (Desk desk) {
-        desk = new Desk(shelf_, token_);
-        return desk;
+contract TrancheManagerFab {
+    // note: this is the mock TrancheManager, which will interface with the lender/tranche side of Tinlake, and does not require auth for now.
+    function newTrancheManager(address shelf_, address token_) public returns (TrancheManager trancheManager) {
+        trancheManager = new TrancheManager(shelf_, token_);
+        return trancheManager;
```

Comment:
In the end, it is the tranche which actually would require the transaction.

We need to pass two addresses in the current setup to the shelf